src/ai/backend/manager/api/resource.py

```diff
@@ -605,74 +613,86 @@ async def get_time_binned_monthly_stats(request: web.Request, user_uuid=None):
     # Build time-series of time-binned stats.
     now_ts = now.timestamp()
     start_date_ts = start_date.timestamp()
-    ts = start_date_ts
-    tseries = []
-    # Iterate over each time window.
-    while ts < now_ts:
-        # Initialize the time-binned stats.
-        num_sessions = 0
-        cpu_allocated = 0
-        mem_allocated = 0
-        gpu_allocated = Decimal(0)
-        io_read_bytes = 0
-        io_write_bytes = 0
-        disk_used = 0
-        # Accumulate stats for containers overlapping with this time window.
-        for row in rows:
-            if (
-                ts + time_window <= row.created_at.timestamp()
-                or ts >= row.terminated_at.timestamp()
-            ):
-                continue
-            num_sessions += 1
-            cpu_allocated += int(row.occupied_slots.get("cpu", 0))
-            mem_allocated += int(row.occupied_slots.get("mem", 0))
-            if "cuda.devices" in row.occupied_slots:
-                gpu_allocated += int(row.occupied_slots["cuda.devices"])
-            if "cuda.shares" in row.occupied_slots:
-                gpu_allocated += Decimal(row.occupied_slots["cuda.shares"])
-            raw_stat = await redis_helper.execute(
-                root_ctx.redis_stat, lambda r: r.get(str(row["id"]))
-            )
-            if raw_stat:
-                last_stat = msgpack.unpackb(raw_stat)
-                io_read_bytes += int(nmget(last_stat, "io_read.current", 0))
-                io_write_bytes += int(nmget(last_stat, "io_write.current", 0))
-                disk_used += int(nmget(last_stat, "io_scratch_size/stats.max", 0, "/"))
-        stat = {
-            "date": ts,
+    time_series_list: list[dict[str, Any]] = [
+        {
+            "date": idx * time_window + now_ts,
```

Comment:
`date` calculation is incorrect. The `date` is starting from `now_ts` in this logic, but this should start from one month ago. Please refer to the previous logic, where `ts` starts from 30 days ago: https://github.com/lablup/backend.ai/pull/1214/files?w=1#diff-51adf4e1db91582bd045aea8a34e9565383e2c437c8cf7aa9eab64aebbdf9b3cL643

Reply:
fix this part. thank you!